src/main/java/frc/robot/commands/AllianceColor.java

```diff
@@ -0,0 +1,49 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj.DriverStation;
+import edu.wpi.first.wpilibj.DriverStation.Alliance;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.StripSubsystem;
+import frc.robot.subsystems.StripSubsystem.Pattern;
+
+public class AllianceColor extends CommandBase {
+
+  private Alliance alliance;
+  private StripSubsystem strip;
+
+  public AllianceColor() {
+    strip = StripSubsystem.getInstance();
+    addRequirements(strip);
+  }
+
+  @Override
+  public void initialize() {
+    if (alliance != null) {
```

Comment:
what if alliance _is_ null? You would want to set it to `DriverStation.getInstance().getAlliance()`, but does that happen?

Reply:
But you see... in the command for wpi lib, It is already dealt with. Null makes it set to invalid which in then does the else which makes the lights sad and grey. 
![image](https://user-images.githubusercontent.com/46285642/156078201-f71ca23b-0287-4d66-bb1e-8f03724d4677.png)